apps/api/src/app/auth/auth.module.ts

```diff
@@ -22,7 +22,7 @@ import { TwitterStrategy } from "./strategies/twitter.strategy"
     providers: [
         GithubStrategy,
         TwitterStrategy,
-        RedditStrategy,
+        // RedditStrategy,
```

Comment:
Why is this line commented?

Reply:
Oh, I just did this because there is no REDDIT_KEY in env yet and to run the api. I'll put it back